migrations/3_deploy.js

```diff
@@ -100,6 +102,10 @@ module.exports = async function(deployer, network, accounts) {
       { Contract: TTokenRegistry, name : logicNames.TTokenRegistry },
     ];
 
+    const loanLib = await LoanLib.new();
```

Comment:
Why don't we use the deployerApp.deploy(...) for libraries?

Also, there is a `deployerApp.links(contract, libraries)` function. So, the deployer app stores the addresses (including the library)